exposee-ui/src/components/Mainpage/Main.css

```diff
@@ -1,30 +1,59 @@
 .main {
-    max-width: 800px;
-    margin: auto;
-    padding: 20px;
-  }
-  
-  .header {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    margin-bottom: 20px;
-  }
-  
-  .user-info {
-    display: flex;
-    align-items: center;
-    gap: 10px;
-  }
-  
-  .user-info a {
-    margin-top: 10px;
-    margin-right: 20px;
-  }
-  
-  .user-info button {
-    color: blue;
-    cursor: pointer;
-    text-decoration: underline;
-    margin-right: 10px; /* Adjust the margin value as needed */
-  }
\ No newline at end of file
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 10px;
+  background-color: #f0f0f0;
+  margin-bottom: 20px;
+}
+
+.user-info {
+  display: flex;
+  align-items: center;
+}
+
+.video-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-gap: 20px;
+}
+
+.video-container {
+  border-radius: 20px;
+  overflow: hidden;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  transition: transform 0.3s ease;
+}
+
+.video-container:hover {
+  transform: scale(1.05);
+}
+
+.video-container.active {
+  border: 2px solid #007bff;
+}
+
+video {
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+  border-radius: 20px 20px 0 0;
+}
+
+p {
```

Comment:
it's generally best practice to give your html elements class names to describe what the element is. eg .video-container is great!

Reply:
I don't think the "video" and "p" settings actually do anything in this file. If they don't you can just remove them